### PR TITLE
SCrypt support

### PR DESCRIPTION
**Pull request by tworec**
To support geth keyfiles

### CODE REVIEW

File: src/keyfile.rs [hunk 1]
```diff
@@ -35,10 +35,9 @@ pub struct Crypto {
     pub cipherparams: Aes128Ctr,
     /// Cipher bytes
 	pub ciphertext: Bytes,
```
Comment (tomusdrw) [inline src/keyfile.rs]: Do you mind reformatting the whole file? Just noticed that there are some issues

File: src/keyfile.rs [hunk 2]
```diff
@@ -60,10 +59,12 @@ pub struct Aes128Ctr {
 
 /// Key-Derivation function
 #[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase")]
+#[serde(rename_all = "camelCase", tag = "kdf", content = "kdfparams")]
 pub enum Kdf {
     /// Password-based KDF 2
-	Pbkdf2,
+	Pbkdf2(Pbkdf2),
```
Comment (tomusdrw) [inline src/keyfile.rs]: Can you fix the indentation here?

**Review by tomusdrw — APPROVED** [review]
Awesome, thank you!

File: src/keyfile.rs (hunk 2 above)
Comment (tworec) [reply src/keyfile.rs]: what about `cargo fmt`?

**Review by tomusdrw — APPROVED** [review]

**Comment by tworec** [comment]
pls publish to crates.io

**Comment by tomusdrw** [comment]
@tworec already done.

**Pull request merged**